### DISCUSSION TITLE
What is in your opinion the most important thing to fix, add or remove in projectfnf

### DISCUSSION THREAD

**Discussion by aflacc** [aflacc/ProjectFNF#38] [Development] [answered]
I'm just curious what I should do

**Comment by VR-Core** [comment]
Fix the input system

**Comment by aflacc** [comment]
> Fix the input system

god dammit core

**Comment by aflacc** [comment]
i told you to shut up about the inputs im working on it okay

**Comment by aflacc** [comment]
grrrrgrrrgrgrgrgrrr

**Comment by VR-Core** [comment]
Just do it

**Comment by Heyitstoes** [comment]
fix the input system

**Comment by aflacc** [comment]
AAAAAAAAAAAAAAAAAAAAAAAAA

**Comment by VR-Core** [comment]
Fix the input system

**Comment by VR-Core** [comment]
It would be funny and I would stop yelling at you

**Comment by aflacc** [comment]
I AM WORKING ON IT

**Comment by Heyitstoes** [comment]
work harder

**Comment by aflacc** [comment]
NO

**Comment by VR-Core** [comment]
Yes

**Comment by aflacc** [comment]
do you want me to get burned out

**Comment by aflacc** [comment]
just like core did with vs aflac

**Comment by VR-Core** [comment]
This is how it feels when you rush me on stuff Aflac

**Comment by aflacc** [comment]
yes I know i realize that now

**Comment by VR-Core** [comment]
Perfect

**Comment by VR-Core** [comment]
I’ll stop annoying you now

**Comment by Heyitstoes** [comment]
remove herobrine

**Comment by VR-Core** [comment]
REMOVE HEROBRINE

**Comment by aflacc** [comment]
no more herobrine!!

**Comment by VR-Core** [comment]
GASP

**Comment by aflacc** [comment]
gdnjfgndkgrnvajsneovmutbhigobsnihtgdfnhvjkltrne

**Comment by aflacc** [comment]
so anyways pog

**Comment by aflacc** [comment]
also core I finished the youtube video did you see it

**Comment by VR-Core** [comment]
Fuck discord’s dms we usin github suggestions

**Comment by Heyitstoes** [comment]
aflac when are you making the tutorial for project fnf

**Comment by aflacc** [comment]
hell yeah

**Comment by aflacc** [comment]
all my homies hate discord github issues is my fren now

**Comment by Heyitstoes** [comment]
beep

**Comment by aflacc** [comment]
beep

**Comment by Heyitstoes** [comment]
_**beep**_

**Comment by aflacc** [comment]
**_beep_**

**Comment by Heyitstoes** [comment]
_**BEEP**_

**Comment by aflacc** [comment]
### _**beep**_

**Comment by aflacc** [comment]
we have returned to monke

**Comment by aflacc** [comment]
oaoaooaoaoaoao

**Comment by UniversePM** [comment]
INPUT SYSTEM

**Comment by aflacc** [comment]
oh my god

I'm gonna say this once more and then I'm not repeating myself.
I have an input system in development by a friend and we are going to use it. There is also a good chance we will just be using the Week 7 input system. Please stop asking for the input system. We will get one when we get one.

**Comment by Heyitstoes** [comment]
Well Aflac, you can't really blame them. A broken input system ruins the experience for the mods they create with YOUR engine. You can't get mad at them for a big bug that you haven't fixed.

**Comment by ghost** [comment, marked as answer]
Revamped debug menu AND the cursor disappears after closing the debug menu

**Comment by aflacc** [comment]
> Well Aflac, you can't really blame them. A broken input system ruins the experience for the mods they create with YOUR engine. You can't get mad at them for a big bug that you haven't fixed.

Yeah but I already said we getting an input system. They aren't that easy to make. I'm not just taking kades inputs because I want this engine to be unique. Modcharts will likely stay the only thing from KE that will be in here. ProjectFNF isn't also currently built exactly for hard mods, but more for just visually appealing ones. Also I am trying to make it super flexable. I plan to make it so you can paste mod  assets into a folder to apply them to the game without needing the source, allowing you to combine multiple mods into one .exe build. I do understand you guys want a new input system and I get it the current one sucks. But I'm not the one making the inputs. I dunno if the person whos making it is actually here (in the discord or just sees this in general) but if you do see this feel free to step up and tell them

**Comment by aflacc** [comment]
hippity hoppity this is now discussion tabs propety

**Comment by aflacc** [comment]
who wants to have their message marked as the answer

**Reply by ghost** [reply]
Me? Mine's a good idea that's not the input system

**Reply by aflacc** [reply]
send the message not as a reply
